.github/workflows/ci.yml

```diff
@@ -22,6 +20,7 @@ on:
   pull_request:
     branches:
       - master
```

Comment:
we still get PRs to release branches, so the normal workflow still needs to run

Reply:
if the two are incompatible, perhaps you also need a separate workflow for release-2020.06 PRs?

Reply:
This should still run for PRs and deployments